lib/pbench/server/api/__init__.py

```diff
@@ -23,11 +23,11 @@
 from pbench.server.api.resources.query_apis.datasets_list import DatasetsList
 from pbench.server.api.resources.query_apis.datasets_publish import DatasetsPublish
 from pbench.server.api.resources.query_apis.elasticsearch_api import Elasticsearch
-from pbench.server.api.resources.query_apis.metadata_index.index_mappings import (
-    IndexMappings,
+from pbench.server.api.resources.query_apis.datasets.dataset_mappings import (
+    DatasetMappings,
 )
-from pbench.server.api.resources.query_apis.index_search import IndexSearch
-from pbench.server.api.resources.query_apis.metadata_index.namespace_and_rows import (
+from pbench.server.api.resources.query_apis.dataset_summary import DatasetSummary
+from pbench.server.api.resources.query_apis.datasets.namespace_and_rows import (
     SampleNamespace,
     SampleValues,
 )
```

Comment:
Similarly, once we're settled on the names, it would probably be worth re-ordering the `api.add_resource()` calls, as well.  (Perhaps you could merge two commits:  one with the name changes, and a separate one with the reorderings.)